composer.json

```diff
@@ -175,6 +175,9 @@
         "drupal/simple_sitemap": "^2.8",
         "drupal/redirect": "^1.0@alpha",
         "drupal/metatag": "^1.0",
+        "drupal/webform": "^5.0@beta",
+        "drupal/captcha": "^1.0@beta",
+        "drupal/recaptcha": "^2.2"
```

Comment:
Comma is missed at the end of the line. Please add comma and re-generate `composer.lock`.